cmd/mothership/mothership/start/http.go

```diff
@@ -353,6 +357,48 @@ func getReconciliations(o *Options, w http.ResponseWriter, r *http.Request) {
 		})
 	}
 
+	//filter by time
+	if after := r.URL.Query().Get(paramAfter); after != "" {
```

Comment:
Would not do it in code, let the DB do that for you by setting the needed WHERE condition. You can set